multicluster/apis/multicluster/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -44,14 +44,6 @@ func (in *ClusterClaim) DeepCopy() *ClusterClaim {
 	return out
 }
 
-// DeepCopyObject is an autogenerated deepcopy function, copying the receiver, creating a new runtime.Object.
-func (in *ClusterClaim) DeepCopyObject() runtime.Object {
```

Comment:
DeepCopyObject() is required, otherwise you will get similar error like below, so I am afraid you need to revert the change to this file.
```
cannot use clusterClaimList (variable of type *v1alpha1.ClusterClaimList) as client.ObjectList value in argument to r.List: missing method DeepCopyObject
```

Reply:
This should be fixed when we removed controller-gen to generate the deepcopy.go file